src/views/homepage/components/learn-section/index.tsx

```diff
@@ -55,7 +55,7 @@ export default function LearnSection({
             } as CSSProperties
           }
         >
-          {tutorials.map((tutorial) => {
+          {tutorials.map((tutorial: TutorialCardProps) => {
```

Comment:
Where is this tutorials data coming from 🤔 ?

Reply:
Oh I see it's hardcoded. nvm

Reply:
Hard-coded in the homepage JSX! I figured for this PR we're not gonna change that.

Reply:
For sure! Just a reminder for me that I need to fix it ha

Reply:
Oops, crossed wires there - didn't see you post that yet 😅 ⚡